openqaoa/problems/converters.py

```diff
@@ -14,11 +14,11 @@
 
 import numpy as np
 from collections import defaultdict
-from .problem import QUBO
+from .qubo import QUBO
 from typing import Union
 
 
-class FromDocplex2IsingModel:
+class FromDocplex2IsingModel(object):
```

Comment:
Is this correct?

Reply:
I usually prefer classes being more verbose about its inheritance.
Its technically only used in python2 and for reverse compatibility. (Object) is automatically added by the interpreter so its unnecessary. Let me know if u think we should remove it. We should also consider it as a future standard for writing objects

Reply:
If unnecessary maybe I will remove it. @vishal-ph ?

Reply:
let's remove it. Yes its a python 2 thing

Reply:
Although its more cosmetic, we agreed that the difference between the 2 would be if `(object)` is included. The following class would have initialisation as part of its design. If we do not intend to initialise the class, i.e. the class only contains methods, we will not include `(object)`.